src-electron/db/db-mapping.js

```diff
@@ -446,6 +446,13 @@ exports.map = {
       dirty: x.DIRTY == 1,
     }
   },
+  upgrade: (x) => {
+    if (x == null) return undefined
+    return {
+      upgrade: x.UPGRADE,
```

Comment:
You can parse here via dbApi.fromDbBool(x.ATTRIBUTE_NAME), similar to other functions  and avoid parsing again in getSessionUpgradeFlag()

Reply:
okay thanks